### PR TITLE
Add RoleName to headers in ping requests to QuickPulse

### PR DESCRIPTION
**Pull request by rambhatt-msft**
Fix Issue # .

## Changes
(Please provide a brief description of the changes here.)

### Checklist
- [x] I ran Unit Tests locally.
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue if available.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed

The PR will trigger build, unit tests, and functional tests automatically. Please follow [these](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) instructions to build and test locally.

### Notes for authors:
- FxCop and other analyzers will fail the build. To see these errors yourself, compile localy using the Release configuration.

### Notes for reviewers:

- We support [comment build triggers](https://docs.microsoft.com/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)
  - `/AzurePipelines run` will queue all builds
  - `/AzurePipelines run <pipeline-name>` will queue a specific build

### CODE REVIEW

File: WEB/Src/PerformanceCollector/Perf.Tests/QuickPulse/QuickPulseServiceClientTests.cs [hunk 1]
```diff
@@ -2115,6 +2115,7 @@ private void ProcessRequest(HttpListener listener, AutoResetEvent ev)
                                 var invariantVersion =
                                     context.Request.Headers[QuickPulseConstants.XMsQpsInvariantVersionHeaderName];
                                 var streamId = context.Request.Headers[QuickPulseConstants.XMsQpsStreamIdHeaderName];
+                                var roleName = context.Request.Headers[QuickPulseConstants.XMsQpsRoleNameHeaderName];
```
Comment (tokaplan) [inline WEB/Src/PerformanceCollector/Perf.Tests/QuickPulse/QuickPulseServiceClientTests.cs:2118]: Are you verifying the value anywhere?

Comment (rambhatt-msft) [reply WEB/Src/PerformanceCollector/Perf.Tests/QuickPulse/QuickPulseServiceClientTests.cs:2120]: Yes, its in the _QuickPulseServiceClientSubmitsStreamIdToServiceWithPing_ method

**Review by tokaplan — APPROVED** [review]

**Review by tokaplan — APPROVED** [review]

**Comment by TimothyMothra** [comment]
Please leave a comment in the Changelog, under the VNEXT header

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿# Changelog
 
 ## VNext
-
+- Add RoleName as a header with Ping Reuests to QuickPulse
```
Comment (TimothyMothra) [inline CHANGELOG.md:4]: please include the link. same as your previous change on line 7

**Review by TimothyMothra — APPROVED** [review]

**Pull request merged**